libs/datamodel/connectors/dml/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+//! This module contains the models representing the Datamodel part of a Prisma schema.
+//! It contains the main data structures that the engines can build upon.
+
+pub mod datamodel;
+pub mod default_value;
+pub mod r#enum;
+pub mod field;
+pub mod model;
+pub mod native_type_constructor;
+pub mod relation_info;
+pub mod scalars;
+pub mod traits;
```

Comment:
I think we should follow the same pattern as with the predecessor of this file. So just use `mod` and then `pub use` everything. That makes it easier for consumers of this crate and allows easy restructuring of the internal file structure.